docs/advanced/experimental/kubernetes.md

```diff
@@ -63,6 +63,24 @@ Forwarding from [::1]:8080 -> 8080
 embedding size:  512                                                                                       
 ```
 
+You might have noticed that the above `Flow` has been deployed to the Namespace `example-clip`.
+By default, we use the `Flow` name as the `Kubernetes` namespace.
```

Comment:
I would add a sentence that we heavily recommend to deploy ` Flow` in a separate K8s namespace. It does not have to have the Flow's name, but it probably should not run other services unless the user really knows what they are doing. We are doing a lot of things in the background in the cluster, like changing security permissions.

Reply:
added to doc top